js/login.js

```diff
@@ -0,0 +1,65 @@
+async function sendAuthRequest() {
```

Comment:
Add comments to everything pls

Reply:
@Kuan-Zher Added code documentation for the functions here too